src/__tests__/operate.test.js

```diff
@@ -0,0 +1,9 @@
+import operate from '../logic/operate';
+
+describe('operate unit', () => {
+  const result = operate('1', '3', '+');
+
+  test('1 + 3 should be equale to 4', () => {
+    expect(result).toBe('4');
+  });
```

Comment:
- [ ] Please make sure to test all the operations (-, +, /, *) and the other cases when you the user want to reset the calculator